src/View/Helper/SchemaHelper.php

```diff
@@ -100,6 +100,28 @@ protected function customControlOptions($name) : array
                     ],
                 ];
 
+            case 'username':
+                return [
+                    'class' => 'email',
+                    'readonly' => true,
+                ];
+
+            case 'email':
```

Comment:
not `readonly` in general `email ` is used in `users` and `profiles` where it may be changed